js/ui/components/clubhouse.js

```diff
@@ -276,9 +345,8 @@ var ClubhouseNotificationBanner = new Lang.Class({
     },
 
     _setupNextPageButton: function() {
-        let button = new St.Button({ style_class: 'notification-button',
+        let button = new St.Button({ style_class: 'notification-button-next',
```

Comment:
A more typical way to implement this would be to add a second style class, instead of overriding the initial one.

Reply:
+1